Drink_Book_PR/DrinkBook/User.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DrinkBook
+{
+    public class User
+    {
+        private string name;
+        private int age;
+        private int budget;
+        private int drinkableWhiskeys;
+        private int drinkableVodkas;
+        private string preferredMusic;
+
+        public string Name
+        {
+            get
+            {
+                return name;
```

Comment:
`this.name` or return simple constructor as per Tanya's suggestion: You can make use of the Auto-Implemented properties. Ref: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties